servers/quarkus-server/build.gradle.kts

```diff
@@ -39,6 +39,7 @@ dependencies {
   implementation(project(":nessie-model"))
   implementation(project(":nessie-services"))
   implementation(project(":nessie-quarkus-common"))
+  implementation(project(":nessie-events-quarkus"))
```

Comment:
Wonder whether and how `nessie-quarkus-cli` needs to be updated as well.

Reply:
Nessie CLI does not depend on quarkus-server and hence does not have any event-related stuff. IOW, the `VersionStore` bean won't produce any events. I thought that was fine since it's a CLI devoted to repository maintenance. Do you think we should add events there as well?

Reply:
Yea, I think that's fine then.